eng/scripts/Invoke-GeneratAndBuildV2.ps1

```diff
@@ -0,0 +1,87 @@
+#Requires -Version 7.0
+param (
+  [string]$inputJsonFile,
+  [string]$outputJsonFile
+)
+
+. (Join-Path $PSScriptRoot automation GenerateAndBuildLib.ps1)
+
+$inputJson = Get-Content $inputJsonFile | Out-String | ConvertFrom-Json
+$swaggerDir = $inputJson.specFolder
+$swaggerDir = Resolve-Path $swaggerDir
+$swaggerDir = $swaggerDir -replace "\\", "/"
+[string[]] $inputFilePaths = $inputJson.changedFiles;
+$readmeFiles = $inputJson.relatedReadmeMdFiles
+$commitid = $inputJson.headSha
+$repoHttpsUrl = $inputJson.repoHttpsUrl
+$downloadUrlPrefix = $inputJson.installInstructionInput.downloadUrlPrefix
+[string] $autorestConfig = $inputJson.autorestConfig
+
+$autorestConfigYaml = ""
+if ( $null -ne $autorestConfig -and $autorestConfig -ne "") {
+    $autorestConfig | Set-Content "config.md"
+    $autorestConfigYaml = Get-Content -Path .\config.md
+    $range = ($autorestConfigYaml | Select-String -Pattern '```').LineNumber
+    if ( $range.count -gt 1) {
+        $startNum = $range[0];
+        $lines = $range[1] - $range[0] - 1
+        $autorestConfigYaml = ($autorestConfigYaml | Select -Skip $startNum | Select -First $lines) |Out-String
+    }
+    Install-Module -Name powershell-yaml -Force -Verbose -Scope CurrentUser
+    Import-Module powershell-yaml
```

Comment:
Please use some of our module install helpers. See [import example](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1#L6) and [usage example](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1#L260). Running `Install-Module` even if it's already installed adds a lot of unnecessary time to the script, e.g. when testing locally.

Reply:
Thanks @benbp . I updated accordingly.